app/ai/voice/agents/breeze_buddy/features/summarizer/context_summarizer.py

```
@@ -0,0 +1,237 @@
+        )
+
+    def add_message(self, message: ChatCompletionMessageParam):
+        """Adds a message to the context and increments the turn count if it's a user message."""
+        super().add_message(message)
+        message_dict = cast(Dict[str, Any], message)
+        if message_dict.get("role") == "user":
+            self._turn_count += 1
+            logger.debug(
+                f"--- Breeze Buddy Summarizer: Turn count incremented to: {self._turn_count} ---"
+            )
+            asyncio.create_task(self._check_if_summary_needed())
```

Ruff (0.15.0)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)